samples/flattened_theia-nodejs.yaml

```diff
@@ -11,20 +11,15 @@ spec:
         git:
           remotes:
```

Comment:
I think it makes sense to rename the file. it's not theia nodejs anymore, just theia next

Reply:
done!